cmd/config.go

```diff
@@ -59,13 +61,15 @@ type Wildcard struct {
 const (
 	defaultMaximumGitLabAPIRequestsPerSecond      = 10
 	defaultOnInitFetchRefsFromPipelinesDepthLimit = 100
-	defaultPipelinesMaxPollingIntervalSeconds     = 3600
 	defaultPipelinesPollingIntervalSeconds        = 30
 	defaultProjectsPollingIntervalSeconds         = 1800
 	defaultRefsPollingIntervalSeconds             = 300
+
+	errNoProjectsOrWildacrdConfigured = "you need to configure at least one project/wildcard to poll, none given"
```

Comment:
There is a misspell in the variable name and I like the Thanos approach of not setting it as a variable if it is only used once. It reduced brain cycles in my opinion : https://thanos.io/contributing/coding-style-guide.md/#avoid-defining-variables-used-only-once

Reply:
fixed, thanks 😄 weird that `make lint` didn't catch it